### PR TITLE
ci(actions): Bump actions to resolve CI warnings

### PR DESCRIPTION
**Pull request by tony**
[Example job](https://github.com/getsentry/responses/actions/runs/3329947398)

<details>

<summary>Screenshot</summary>

image

</details>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,11 +15,11 @@ jobs:
       fail-fast: false
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
       with:
         fetch-depth: 1
 
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v3
```
Comment (beliaev-maksim) [inline .github/workflows/ci.yml:22]:
```suggestion
    - uses: actions/setup-python@v4
```

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tony) [reply .github/workflows/ci.yml]: Good catch! Updated via rebase

**Review by beliaev-maksim — APPROVED** [review]

**Comment by tony** [comment]
@beliaev-maksim I added one more bump via squash, @actions/checkout v2 -> v3. [job](https://github.com/getsentry/responses/actions/runs/3455737130),  [release notes](https://github.com/actions/checkout/releases/tag/v3.0.0)

**Comment by markstory** [comment]
Thank you 🎉

**Pull request merged**